language-support/ts/codegen/tests/daml/Main.daml

```diff
@@ -0,0 +1,53 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+daml 1.2
+module Main where
+
+import DA.TextMap
+import Lib.Mod
+
+data AllTypes = AllTypes {
+  unit: ();
```

Comment:
Congratulations, you have won the award for the weirdest record syntax that I’ve seen in DAML so far :trollface: